parser_library/test/lsp/lsp_completion.cpp

```diff
@@ -73,3 +73,28 @@ TEST(lsp_completion, completion_list_empty)
 
     EXPECT_TRUE(result.empty());
 }
+
+TEST(lsp_completion, macro_operands)
+{
+    const std::string input = R"(
+         MACRO
+         MAC   &A,&B=(DEFAULTB,                                        X
```

Comment:
Not sure if this can be done easily but I can imagine that the completion list would be updated based on already used parameters. 

E.g. when I write  `    MAC &A,` I would expect the completion list to consist only of the remaining keyword parameter `&B`